src/components/Pagination.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { PaginationStyle } from './styled/Pagination.styled'
+import { FaChevronLeft, FaChevronRight } from 'react-icons/fa'
+
+const Pagination = () => {
+  return (
+    <PaginationStyle>
```

Comment:
Pagination Style needs more work, lacking previous and next buttons, also styling need to be improved current choice of color makes it barely readable.